modules/hardware/nvidia-jetson-orin.nix

```diff
@@ -10,8 +10,32 @@
 
   nixpkgs.hostPlatform.system = "aarch64-linux";
 
-  boot.loader = {
-    efi.canTouchEfiVariables = true;
-    systemd-boot.enable = true;
+  boot.kernelPatches = [
+    {
+      name = "passthrough-patch";
+      patch = ./pci-passthrough-test.patch;
+    }
+  ];
+
+  hardware.deviceTree = {
+    enable = true;
+    name = "tegra234-p3701-host-passthrough.dtb";
+  };
+
+  # Passthrough Jetson Orin WiFi card
+  boot.kernelParams = [
+    "vfio-pci" "ids=10ec:c82f"
```

Comment:
I'd like to have some toggle for this, and not enable this always by default

Reply:
What would be the motivation on Orin reference device? Wifi is basically the only networking interface we can easily and consistently pass through to netvm. I'd rather have it enabled by default even when we can't/shouldn't make it connect any WIFI automatically. And to support wifi, bring `nmcli` to `netvm` to easily scan and connect wifis over debug console.